examples/pxScene2d/src/optimus_client.cpp

```diff
@@ -86,7 +86,7 @@ namespace OptimusClient
     fflush(stdout);
     if (gSceneContainer.getPtr())
     {
-      rtLogInfo("scene container set, attempting to get service");
+      rtLogDebug("scene container set, attempting to get service");
```

Comment:
Let's leave this log statement as is because it's rarely logged and it's helpful for debugging.